x-pack/plugins/ml/public/application/model_management/add_model_flyout.tsx

```diff
@@ -0,0 +1,445 @@
+                                          </EuiText>
+                                        </EuiFlexItem>
+                                        {model.recommended ? (
+                                          <EuiFlexItem grow={false}>
+                                            <EuiToolTip
+                                              content={
+                                                <FormattedMessage
+                                                  id="xpack.ml.trainedModels.modelsList.recommendedDownloadContent"
+                                                  defaultMessage="Recommended ELSER model version for your cluster's hardware configuration"
+                                                />
+                                              }
+                                            >
+                                              <EuiBadge color="hollow">
+                                                <FormattedMessage
+                                                  id="xpack.ml.trainedModels.addModelFlyout.recommendedDownloadLabel"
+                                                  defaultMessage="Recommended"
+                                                />
+                                              </EuiBadge>
+                                            </EuiToolTip>
+                                          </EuiFlexItem>
+                                        ) : null}
+                                      </EuiFlexGroup>
+                                    }
+                                    name={model.model_id}
+                                    value={model.model_id}
+                                    checked={model.model_id === selectedModelId}
+                                    onChange={() => setSelectedModelId(model.model_id)}
+                                  />
+                                  <EuiSpacer size="m" />
+                                </React.Fragment>
+                              );
+                            })}
+                          </EuiFormFieldset>
+                        </React.Fragment>
+                      );
+                    }
+                  )}
+                  <EuiButton
+                    onClick={onSubmit.bind(null, selectedModelId!)}
+                    fill
+                    disabled={!selectedModelId}
+                  >
+                    <FormattedMessage
+                      id="xpack.ml.trainedModels.addModelFlyout.downloadButtonLabel"
+                      defaultMessage="Download"
+                    />
+                  </EuiButton>
+                </>
+              ),
+            },
+          ]
+        : []),
+      {
+        id: 'thirdParty',
+        name: (
+          <FormattedMessage
+            id="xpack.ml.trainedModels.addModelFlyout.thirdPartyLabel"
+            defaultMessage="Third-party"
+          />
+        ),
+        content: (
```

Comment:
It might be worth breaking this static content out into a separate file. To help with readability.